Package.swift

```diff
@@ -1,32 +1,7 @@
-// swift-tools-version:5.4
+// swift-tools-version:5.6
```

Comment:
Why need to update tools version? Can we keep `Package@5.4.swift` for older toolchains?

Reply:
Let me double-check this once again... I've tried this once and 5.4/5.5 didn't really work with latest `carton`